terraform/core/51-load-all-academy-data-into-redshift-serverless.tf

```diff
@@ -0,0 +1,22 @@
+module "load_all_academy_data_into_redshift" {
+  source                         = "../modules/aws-glue-job"
+  is_live_environment            = local.is_live_environment
+  is_production_environment      = local.is_production_environment
+  department                     = module.department_parking_data_source
```

Comment:
this is the wrong department which will mean the glue role won't have access to the s3 locations and the module will look for the script in /scripts/jobs/department/script_name.py - in this case /scripts/jobs/parking/load_all_academy_data_into_redshift.py  - which doesn't exist.